base/deprecated.jl

```diff
@@ -801,6 +801,8 @@ macro MIME(s)
 end
 end)
 
+@deprecate write(x) write(STDOUT::IO, x)
```

Comment:
this should probably go at the end of the file, I don't think we're going to squeeze in any deprecations right now